build.gradle

```diff
@@ -200,10 +200,18 @@ def depTehuti = 'io.tehuti:tehuti:0.7.0'
 
 
 shadowJar {
+    zip64 true
```

Comment:
We are creating a fatjar that may have more than 64K files. In that case zip64 is required.